### PR TITLE
Add speedtest package

### PR DESCRIPTION
**Pull request by pgdagenais**
Add the speedtest-cli package to chromebrew !

https://github.com/sivel/speedtest-cli

### CODE REVIEW

File: packages/speedtest.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+require 'package'
+
+class Speedtest < Package
+  description 'Command line interface for testing internet bandwidth using speedtest.net'
+  homepage 'https://github.com/sivel/speedtest-cli'
+  version '2.0.0'
+  source_url 'https://github.com/sivel/speedtest-cli/archive/v2.0.0.tar.gz'
+  source_sha256 '732daf109a3399c794b293723199d740b0accddef86007a0f85f577bd4ba6c9a'
+
+  depends_on 'python3'
```
Comment (uberhacker) [inline packages/speedtest.rb]: Do you mean python27?  The pip executable is provided by python27.  The pip3 executable is provided by python3.

Comment (pgdagenais) [reply packages/speedtest.rb]: Opps ! I just fixed it.

File: packages/speedtest.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+  def self.install
+    system "yes | pip install setuptools"
```
Comment (uberhacker) [inline packages/speedtest.rb]: Do you need pip3 here also?

Comment (pgdagenais) [reply packages/speedtest.rb]: @uberhacker fixed.. setuptools for python3 was already installed, my bad

**Pull request merged**